packages/core/src/sagas/connectionStatusUpdateSaga.js

```diff
@@ -12,7 +12,7 @@ export default function* () {
   yield whileConnected(observeConnectionStatus);
 }
 
-function* observeConnectionStatus(directLine) {
+function* observeConnectionStatus({ directLine }) {
```

Comment:
this looks out of place. it doesn't look like the shape of the parameter change?

Reply:
now i see it - nevermind!